generators/client/templates/angular/src/test/javascript/e2e/account/account.spec.ts.ejs

```diff
@@ -33,178 +34,159 @@ describe('account', () => {
     let settingsPage: SettingsPage;
     <%_ } _%>
 
-    beforeAll(() => {
-        browser.get('/');
-        browser.waitForAngular();
+    beforeAll(async () => {
+        await browser.get('/');
         navBarPage = new NavBarPage(true);
-        browser.waitForAngular();
```

Comment:
isnt this angular specific? protractor does some special handling for angular so is it ok to remove it?

Reply:
the protractor documentation (https://www.protractortest.org/#/api?view=ProtractorBrowser.prototype.waitForAngular) says that waitForAngular() is already called before each action that's why I removed it, this does not seem to create any trouble